knt-backend/app/models.py

```diff
@@ -2,42 +2,98 @@
 from flask_restful import Resource
 from flask import request
 
+""" USERS """
+
+
 class User(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     first_name = db.Column(db.Text)
     last_name = db.Column(db.Text)
     nickname = db.Column(db.Text)
     password = db.Column(db.String(64))
-    balance = db.Column(db.Numeric)
+    balance = db.Column(db.Text)
```

Comment:
Why is the balance text?

Reply:
Something something Flask doesn't like it when numeric values need to be json-serialized. It is text, however, in the body of a post request I can have the balance be a floating-point number and it will still show up as such during a GET request, and not as a numeric value. I fell down a rabbit hole trying to figure out how to avoid this but I think this could be explored as an issue after the MVP.

Reply:
Interesting